zowe-api.json

```diff
@@ -0,0 +1,63 @@
+{
+    "name": "API Mediation Layer",
+    "defaultDirName": "api-mediation-layer",
+    "defaultHlqSegment": "ZADSMP",
+    "zfsMegabytes": 500,
+
+    "deployment": {
+        "files": {
+            "discovery-service/build/libs/discovery-service-thin.jar": {
+                "target": "components/api-mediation/discovery-service.jar",
+                "binary": true
+            },
+            "gateway-service/build/libs/gateway-service-thin.jar": {
+                "target": "components/api-mediation/gateway-service.jar",
+                "binary": true
+            },
+            "api-catalog-services/build/libs/api-catalog-services-thin.jar": {
+                "target": "components/api-mediation/api-catalog-services.jar",
+                "binary": true
+            },
+            "build/libraries.zip": {
+                "target": "components/api-mediation/lib/libraries.zip",
+                "binary": true
+            }
+        }
+    },
+
+    "shellStartCommand": "echo 'Not supported until variable replacement is implemented by Zowe SDK team'",
```

Comment:
We will implement it. It was one of the feature of `mfaas-auto` :)

Reply:
great!